packages/core/src/browser/style/variables-dark.useable.css

```diff
@@ -1,128 +1,115 @@
-  
-    /* Use these font colors against the corresponding main layout colors.
-       In a light theme, these go from dark to light.
-    */
-  
-    --theia-ui-font-color0: var(--md-grey-100);
-    --theia-ui-font-color1: var(--md-grey-300);
-    --theia-ui-font-color2: var(--md-grey-600);
-    --theia-ui-font-color3: var(--md-grey-700);
-    --theia-ui-font-color4: var(--md-grey-800);
-    --theia-ui-font-color5: var(--md-grey-900);
-
-    /* Special font colors */
-    --theia-ui-icon-font-color: #D4D4D4;
-
-    --theia-ui-bar-font-color0: #EEEEEE;
-    --theia-ui-bar-font-color1: #D4D4D4;
-  
-    /* Use these against the brand/accent/warn/error colors.
-       These will typically go from light to darker, in both a dark and light theme
-     */
-  
-    --theia-inverse-ui-font-color0: rgba(0,0,0,1);
-    --theia-inverse-ui-font-color1: rgba(0,0,0,1.0);
-    --theia-inverse-ui-font-color2: rgba(0,0,0,0.7);
-    --theia-inverse-ui-font-color3: rgba(0,0,0,0.5);
-  
-    /* Content Fonts
-  
-    Content font variables are used for typography of user generated content.
-    */
-  
-    --theia-content-font-size: 13px;
-    --theia-content-line-height: 1.5;
-    --theia-content-font-color0: white;
-    --theia-content-font-color1: white;
-    --theia-content-font-color2: var(--md-grey-500);
-    --theia-content-font-color3: var(--md-grey-700);
-  
-    --theia-code-font-size: 13px;
-    --theia-code-line-height: 17px;
-    --theia-code-padding: 5px;
-    --theia-code-font-family: monospace;
-    --theia-ui-padding: 6px;
-  
-  
-    /* Layout
+  --theia-layout-color0: #1e1e1e;
+  --theia-layout-color1: #222222;
+  --theia-layout-color2: #2e2e2e;
+  --theia-layout-color3: #303030;
+  --theia-layout-color4: #333333;
+  --theia-layout-color5: #404040;
   
-    The following are the main layout colors use in Theia. In a light
-    theme these would go from light to dark.
-    */
-    
-    --theia-layout-color0: var(--md-grey-800); /* #2e2e2e */
-    --theia-layout-color1: #333333;
-    --theia-layout-color2: #2b2b2b;
-    --theia-layout-color3: #1e1e1e;
-    --theia-layout-color4: #404040;
-    --theia-layout-color5: var(--md-grey-800);
-    --theia-layout-color6: #1e1e1e;
-    --theia-layout-color7: #333333;
-    
-  /* Brand/accent */
+  /* Brand colors */
 
   --theia-brand-color0: var(--md-blue-700);
   --theia-brand-color1: var(--md-blue-500);
   --theia-brand-color2: var(--md-blue-300);
   --theia-brand-color3: var(--md-blue-100);
 
+  /* Accent colors (bright to dark): Use these to create contrast to layout colors. */
+
   --theia-accent-color0: var(--md-light-blue-300);
-  --theia-accent-color1: var(--md-light-blue-500);
-  --theia-accent-color2: var(--md-light-blue-700);
-  --theia-accent-color3: var(--md-light-blue-900);
-  --theia-accent-color4: var(--md-grey-800);
-  --theia-accent-color5: var(--md-grey-800);
+  --theia-accent-color1: #28abe7;
+  --theia-accent-color2: #2288be;
+  --theia-accent-color3: #265f8a;
```

Comment:
Maybe, the accent-colors are too greyish now? 
For example in history list the contrast is very low:
<img width="308" alt="screen shot 2018-04-10 at 16 43 28" src="https://user-images.githubusercontent.com/28291421/38564328-36b4fb36-3cdf-11e8-9002-b21dd89764ed.png">

Reply:
Made the font used for commit time a bit darker, and made the accent color brighter for the bright theme.